docs/requirement-sets/outlook/requirement-set-1.12/office.context.md

```diff
@@ -0,0 +1,309 @@
+
+|Requirement| Value|
+|---|---|
+|[Minimum mailbox requirement set version](../outlook-api-requirement-sets.md)| 1.10|
+|[Applicable Outlook mode](/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points)| Compose or Read|
+
+##### Example
+
+```js
+Office.context.auth.getAccessTokenAsync(function(result) {
+    if (result.status === "succeeded") {
+        var token = result.value;
+        // ...
+    } else {
+        console.log("Error obtaining token", result.error);
+    }
+});
+```
+
+<br>
+
+---
+---
+
+#### contentLanguage: String
+
+Gets the locale (language) specified by the user for editing the item.
+
+The `contentLanguage` value reflects the current **Editing Language** setting specified with **File > Options > Language** in the Office client application.
+
+##### Type
+
+*   String
+
+##### Requirements
+
+|Requirement| Value|
+|---|---|
+|[Minimum mailbox requirement set version](../outlook-api-requirement-sets.md)| 1.1|
+|[Applicable Outlook mode](/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points)| Compose or Read|
+
+##### Example
+
+```js
+function sayHelloWithContentLanguage() {
+  var myContentLanguage = Office.context.contentLanguage;
+  switch (myContentLanguage) {
+    case 'en-US':
+      write('Hello!');
+      break;
+    case 'en-NZ':
+      write('G\'day mate!');
+      break;
+  }
+}
+
+// Function that writes to a div with id='message' on the page.
+function write(message){
+  document.getElementById('message').innerText += message;
+}
+```
+
+<br>
+
+---
+---
+
+#### diagnostics: [ContextInformation](/javascript/api/office/office.contextinformation?view=outlook-js-1.12&preserve-view=true)
+
+Gets information about the environment in which the add-in is running.
+
+> [!NOTE]
+> For all Mailbox requirement sets, you can also use the [Office.context.mailbox.diagnostics](/javascript/api/outlook/office.mailbox?view=outlook-js-1.12&preserve-view=true#outlook-office-mailbox-diagnostics-member) property to get similar information.
+
+##### Type
+
+*   [ContextInformation](/javascript/api/office/office.contextinformation?view=outlook-js-1.12&preserve-view=true)
+
+##### Requirements
+
+|Requirement| Value|
+|---|---|
+|[Minimum mailbox requirement set version](../outlook-api-requirement-sets.md)| 1.5|
+|[Applicable Outlook mode](/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points)| Compose or Read|
+
+##### Example
+
+```js
+var contextInfo = Office.context.diagnostics;
+console.log("Office application: " + contextInfo.host);
+console.log("Office version: " + contextInfo.version);
+console.log("Platform: " + contextInfo.platform);
+```
+
+<br>
+
+---
+---
+
+#### displayLanguage: String
+
+Gets the locale (language) in RFC 1766 Language tag format specified by the user for the UI of the Office client application.
+
+The `displayLanguage` value reflects the current **Display Language** setting specified with **File > Options > Language** in the Office client application.
```

Comment:
May also need to update other instances
```suggestion
The `displayLanguage` value reflects the current **Display Language** setting specified with **File** > **Options** > **Language** in the Office client application.
```

Reply:
Thanks for the suggested change. I'll update this for other applicable requirement sets in a separate PR.